src/util/texture.cpp

```diff
@@ -1,31 +1,31 @@
 #include "util/texture.h"
 
-QOpenGLTexture* createTexture(const QImage& image) {
+std::unique_ptr<QOpenGLTexture> createTexture(const QImage& image) {
     if (image.isNull()) {
-        return nullptr;
+        return std::unique_ptr<QOpenGLTexture>{};
```

Comment:
why do you prefer this over just returning `nullptr`?

Reply:
I like to be explicit, but I'm okay with changing that if you commonly return nullptr elsewhere.

Reply:
I'd prefer that.

Reply:
ok, changed back. i think with that I addressed all your comments.